src/main/resources/plugin.yml

```diff
@@ -1,12 +1,12 @@
 name: TradeShop
-version: 2.1.0
+version: 2.2.0
 main: org.shanerx.tradeshop.TradeShop
 api-version: 1.13
 load: POSTWORLD
 
 authors: [Lori00, KillerOfPie]
-description: A new powerful trading plugin.
-website: https://www.spigotmc.org/resources/tradeshop.32762/
+description: A new powerful economy trading plugin based on sign-chest shops.
```

Comment:
Do we want to call it an economy trading plugin? kinda make it sounds like it is built around using currencies...

Reply:
Fair enough, I will change it back

Reply:
i think just taking out economy, the rest is fine, also maybe change the website to tradeshop.github.io